cli/azd/pkg/project/service_manager.go

```diff
@@ -551,6 +551,10 @@ func (sm *serviceManager) GetServiceTarget(ctx context.Context, serviceConfig *S
 func (sm *serviceManager) GetFrameworkService(ctx context.Context, serviceConfig *ServiceConfig) (FrameworkService, error) {
 	var frameworkService FrameworkService
 
+	if serviceConfig.Language == ServiceLanguageNone && serviceConfig.Image != "" {
```

Comment:
I wonder if this deserves a comment that "publishing from image currently shares the same lifecycle implementations as a Docker project". The place I'm coming from is from whether the implementations would ever differ in the future. I could see us refactoring things to not be centralized around "framework services", but rather we simply compose the logical blocks of "a component that pushes to ACR", and "a component that publishes revision to ACA from ACA" to accomplish the task at hand.

Reply:
Will add some comments.